app/src/main/java/com/icebreakers/nexxus/activities/EventListActivity.java

```diff
@@ -356,4 +434,20 @@ public void onMessageEvent(MeetupEvent event) {
         Log.d(TAG, "Checkin-event received " + event);
         checkedInEventListFragment.add(event);
     }
+
+    // UI updates must run on MainThread
+    @Subscribe(sticky = true, threadMode = ThreadMode.MAIN)
+    public void onEvent(String firstName) {
```

Comment:
If its not a lot of work, may onevent should take profileId instead of firstName